libraries/pipes/implementations/rust/quicktype.sh

```diff
@@ -1,4 +1,4 @@
-#!bash
+#!/bin/bash
```

Comment:
What's your motivation behind this change? 

Personally, I'm on NixOS so `/bin/bash` won't work for me

Reply:
Does `#!/usr/bin/env bash` work on NixOS?

Reply:
I change it because #!bash didnt work on my mac. :(

also curious if `env` is working for you.

Reply:
Changed to `/usr/bin/env`. Please tell me if it doesn't works for you. 🙏

Reply:
This seems to be working!